src/main/java/adubbz/nx/loader/common/NXProgramBuilder.java

```diff
@@ -274,6 +276,8 @@ protected void setupRelocations() throws AddressOutOfBoundsException, NotFoundEx
             long pltStart = this.pltEntries.get(0).off;
             long pltEnd = this.pltEntries.get(this.pltEntries.size() - 1).off + 0x10;
             this.memBlockHelper.addSection(".plt", pltStart, pltStart, pltEnd - pltStart, true, false, false);
+            // Disassemble the entire section, so AARCH64PltThunkAnalyzer works.
```

Comment:
I initially didn't include the part where the thunk functions are created, but even after disassembling the entire section the AARCH64PltThunkAnalyzer wasn't able to create the thunk functions.
So I ended up adding that step as well.